baseplate/__init__.py

```diff
@@ -93,8 +96,90 @@ def make_tracing_client(raw_config, log_if_unconfigured=True):
     )
 
 
+def crash_reporter_from_config(raw_config, module_name):
```

Comment:
I'm not at all convinced of taking the module name here. The name is maybe a little nicer than a straight file path, since potentially this function could do fancy stuff down the line with the module name and the `include_paths` directive. Is there a better/nicer way to do this though?